NewHorizons/Builder/Props/GeneralPropBuilder.cs

```diff
@@ -12,65 +12,85 @@ namespace NewHorizons.Builder.Props
 {
     public static class GeneralPropBuilder
     {
-        public static GameObject MakeFromExisting(GameObject go, GameObject planetGO, Sector sector, PropModule.PositionedPropInfo info, bool alignToBody = false, MVector3 normal = null)
+        public static GameObject MakeFromExisting(GameObject go, Transform parent, PropModule.GeneralPointPropInfo info, bool alignToBody = false, MVector3 normal = null, MVector3 defaultPosition = null, string defaultParentPath = null)
         {
+            if (info == null) return go;
+
+            if (info is PropModule.GeneralSolarSystemPropInfo solarSystemInfo && !string.IsNullOrEmpty(solarSystemInfo.parentBody))
+            {
+                var targetPlanet = AstroObjectLocator.GetAstroObject(solarSystemInfo.parentBody);
```

Comment:
does this make things rely on planet build order?

Reply:
It's only used by 'props' in the solar system Vessel config, which gets run after all the planets are built. I can throw in a comment on that